cmd/manager/kodata/knative-eventing/2-upgrade-to-v0.14.0.yaml

```diff
@@ -4,14 +4,17 @@ metadata:
   name: v0.14.0-upgrade
   namespace: knative-eventing
   labels:
-    eventing.knative.dev/release: "v0.14.0"
+    eventing.knative.dev/release: "v0.14.1"
```

Comment:
with the releae label being 0.14.1 - should this file be called `2-upgrade-to-v0.14.`.yaml` ?

Reply:
done.